bin/sass-lint.js

```diff
@@ -32,6 +32,7 @@ program
   .option('-v, --verbose', 'verbose output')
   .option('-f, --format [format]', 'pass one of the available eslint formats')
   .option('-o, --output [output]', 'the path and filename where you would like output to be written')
+  .option('-s, --syntax [syntax]', 'syntax to evaluate the file(s) with (either sass or scss)')
```

Comment:
Not necessarily for now, but may want to look at commander regEx when we redo the CLI in 2.0

https://www.npmjs.com/package/commander#regular-expression